app/services/search_orders.rb

```diff
@@ -15,11 +15,19 @@ def initialize(params, current_user)
   attr_reader :params, :current_user
 
   def fetch_orders
-    @search = search_query.
+    # Remap "billing" address
+    params.dig(:q, :s)&.gsub!("billing_address", "bill_address")
```

Comment:
I just checked out this branch and navigated to `/admin/orders` and got an error:
```
FrozenError in Spree::Admin::OrdersController#index 
can't modify frozen String: "completed_at desc"
```

Looks like we are missing some test coverage here!

Reply:
> Looks like we are missing some test coverage here!

Actually not, lots of specs are failing. I just didn't pay attention because we have so many flaky specs and I assumed that @dacook would have checked the spec outcome. :see_no_evil:

Reply:
This change also broke the sorting function because we have custom search scopes `sort_by_billing_address_name_[asc,desc]` and changing that to `bill` breaks it. Sorry, @dacook, I'll take your commit out again.

Reply:
Sorry, yes it was a bit of an experiment on my part 😬 .